ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -24,7 +24,24 @@
           ...attributes
         />
       </field.Control>
-
+      <Button @variant="secondary" @onClick={{(fn this.handleChange field)}}>
```

Comment:
Shouldn't this be also inside of `<field.Control>`?

Also, and this is really weird, looking at the  button's [template](https://github.com/CrowdStrike/ember-toucan-core/blob/main/ember-toucan-core/src/components/button.hbs), this should be a literal `<button>`. But why am I seeing this as a div in Chrome!? 🤯

<img width="1906" alt="image" src="https://user-images.githubusercontent.com/1325249/228856096-b5ec70ac-6695-47bf-afc2-42e9bc5c7e8c.png">

Reply:
Oh, wait, taking the URL from the preview link (https://678302d9.ember-toucan-core.pages.dev/docs/components/file-input-field), the button is actually a button. 

But it is not on the URL that was in the PR description! 🤔

Reply:
I was having issues with Github actions and publishing the docs yesterday (first Github Actions had problems, then I had a typo in one of my base demos) so maybe that's why....

Reply:
Moved the button into control here: https://github.com/CrowdStrike/ember-toucan-core/pull/108/commits/3bb6d0349bfcfd280691b30045eaefb327736483